portal_sale_personal_data_only/readme/DESCRIPTION.rst

```diff
@@ -1,7 +1,7 @@
-By default, portal users are allowed to see all the orders and invoices in
+By default, portal users are allowed to see all the sale orders in
 which a member of their organization are followers. That could cause a leaking
 of  documents between members and departments and of the organization that
 should stay private.
 
-This module restricts that behaviour so the portal users only see their own
+This module restricts that behavior so the portal users only see their own
```

Comment:
I usually prefer British English over American:

https://grammarist.com/spelling/behavior-behaviour/

but I think the main reason to let it as it was is for reducing the diff.

Reply:
It was just the code spell checker underlining this on red and I simply changed it. Let me undo the change because it is indeed not so important.

OTOH, AFAIK the base lang for odoo is `en_US`. I've never read a guideline on the subject for OCA, but since we follow Odoo's I always understood that's our standard here too. Does it make sense? We need a standard "behavior" in this subject. 🥁 🙃

Reply:
I opened https://github.com/moduon/odoo-community.org/pull/2 as I'm in favor of standardization. However I'll leave it as it is as I'm also in favor of MVP. 😄